WhateverGreen/kern_rad.cpp

```diff
@@ -107,6 +107,9 @@ void RAD::init() {
 
 	// Broken drivers can still let us boot in vesa mode
 	forceVesaMode = checkKernelArgument("-radvesa");
+	
+	// Fix codec PID to be spoofed PID if requested
+	forceCodecInfo = checkKernelArgument("-radcodec");
```

Comment:
If we decide that this needs a boot argument, I request us also add a DeviceProperty with the same purpose.

Reply:
Any reason why `-radvesa` `-rad24` and `-raddvi` do not have DeviceProperty? I used those as a template.

Reply:
-radvesa is a way to boot a failing system. But the others are bugs or rather they were added before we started to implement property mirroring. We should fix them sooner or later. I guess I could handle that myself and also fix your -radcodec.